app/views/records/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>Patient Information for <%= full_name(@record) %></h1>
+<!--h1>Patient Information for <%= full_name(@record) %></h1-->
```

Comment:
good catch, we should probably remove that commented out code as well.  That was removed because it is not redundant.

Old view
<img width="1451" alt="image" src="https://user-images.githubusercontent.com/8173551/50703304-aafa3400-1021-11e9-9750-eff73e7d6408.png">

New View
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/8173551/50703314-b3526f00-1021-11e9-8850-644478c3b27f.png">

Reply:
Removed